src/liballoc/boxed.rs

```diff
@@ -462,22 +481,14 @@ impl<T: ?Sized> Box<T> {
     #[unstable(feature = "box_into_raw_non_null", issue = "47336")]
     #[inline]
     pub fn into_raw_non_null(b: Box<T>) -> NonNull<T> {
-        Box::into_unique(b).into()
-    }
-
-    #[unstable(feature = "ptr_internals", issue = "none", reason = "use into_raw_non_null instead")]
-    #[inline]
-    #[doc(hidden)]
-    pub fn into_unique(b: Box<T>) -> Unique<T> {
-        let b = mem::ManuallyDrop::new(b);
-        let mut unique = b.0;
+        let mut b = mem::ManuallyDrop::new(b);
```

Comment:
Looks like this PR conflicts badly with https://github.com/rust-lang/rust/pull/71168.

Reply:
I’ll deal with it after one of them lands.